### PR TITLE
Migrate to v3 and add touch handling (devices without keyboard or rem…

### PR DESCRIPTION
**Pull request by rafi16jan**
…ote)

### CODE REVIEW

**Comment by angeloanan** [comment]
Whoops, didn't saw this PR's notification. Testing it rn.

**Comment by angeloanan** [comment]
Currently testing the touchscreen feature of the change, some issues show up:
* Swiping left / right too far seem to trigger the browser's built-in go back / forward feature. Is there a way to remove this or make it disappear?
* There is no dedicated swiping area; Page items are selected during the swipe and sometimes, scrollable items are scrolled with it.

I didn't spot any issue with the MV3 upgrade.

With iteration velocity in mind, I suggest splitting the changes into a smaller PR (MV3 & Swipe control). I'll gladly accept the MV3 PR once done!

File: src/chrome/touch_handler.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+gesuredZone.addEventListener('touchend', function(event) {
+    console.log(event);
+    console.log(touchendX = event.changedTouches[0].screenX);
+    console.log(touchendY = event.changedTouches[0].screenY);
+    handleGesure();
+}, false); 
+
+function handleGesure() {
+    var swiped = 'swiped: ';
+    if ((touchendX - touchstartX) <= -130) {
```
Comment (angeloanan) [inline src/chrome/touch_handler.js:23]: Just wondering, where did the number `130` come from? Is it standardized for swipes or is it just a random number that feels "right"?

Comment (rafi16jan) [reply src/chrome/touch_handler.js:23]: >  just a random number that feels "right"

Yes... That's just a threshold that I found when randomly testing it, I'm open to suggestion though, you can test it yourself

**Comment by rafi16jan** [comment]
> * Swiping left / right too far seem to trigger the browser's built-in go back / forward feature. Is there a way to remove this or make it disappear?

[This stackoverflow question](https://stackoverflow.com/questions/15829172/stop-chrome-back-forward-two-finger-swipe) should work but I haven't tested it.

**Comment by angeloanan** [comment]
As there is no further changes from you, to save time, I'm gonna take over this PR and merge the MV3 changes.
Feel free to make another PR covering the touchscreen swipe if you've figured out on how to solve the issues above!

**Pull request closed without merging**